src/writer.js

```diff
@@ -55,11 +55,24 @@ class Writer {
     	return 'data:audio/midi;base64,' + this.base64();
     }
 
-
- 
+		/**
+		 * Output to stdout
+		 *
+		 */
     stdout() {
     	return process.stdout.write(new Buffer(this.buildFile()));
     }
+
+		/**
+		 * Save to MIDI file
+		 *
+		 */
+		 saveMIDI(filename) {
```

Comment:
Hey thanks a lot for this PR.  You mind adjusting the indentation here to use tabs?